addon/modifiers/scroll-into-view.js

```diff
@@ -52,6 +86,17 @@ function scrollIntoView(element, positional, named = {}) {
           left,
         });
       }
+      if (shouldFocusAfterScroll) {
+        let focusElement;
+        if (typeof focusSelector === 'string') {
+          focusElement = element.querySelector(focusSelector);
+        }
+        focusElement = focusElement ?? element.querySelector(DEFAULT_FOCUSABLE_ELEMENTS.join(', '));
```

Comment:
Nit: It might be helpful to add a comment here to explain that this line is allowing us to target the first focusable element.